### PR TITLE
EVG-3197 Run repotracker catchup when confirming base revision

### PR DESCRIPTION
**Pull request by bsamek**

### CODE REVIEW

File: service/project.go [hunk 1]
```diff
@@ -547,6 +547,10 @@ func (uis *UIServer) setRevision(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	// run the repotracker for the project
+	j := units.NewRepotrackerJob(fmt.Sprintf("catchup-set-revision-%s", ts), projectRef.Identifier)
+	j.Run(context.Background())
```
[the remark this answers is not in the document]
Comment (bsamek) [reply service/project.go]: Done

File: service/project.go [hunk 2]
```diff
@@ -547,6 +548,15 @@ func (uis *UIServer) setRevision(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	// run the repotracker for the project
+	ts := util.RoundPartOfHour(0).Format("2006-01-02.15-04-05")
+	j := units.NewRepotrackerJob(fmt.Sprintf("catchup-set-revision-%s", ts), projectRef.Identifier)
```
Comment (john-m-liu) [inline service/project.go]: Do we want to use the same identifier pattern as the regular catchup job so that we don't run 2 of them at the same time?

Comment (bsamek) [reply service/project.go]: doen

Comment (bsamek) [reply service/project.go]: done

Comment (john-m-liu) [reply service/project.go]: I think you need to use a similar time rounding format as the job for this to work. The existing job rounds to 30 mins, but this one can probably round to 5?

Comment (bsamek) [reply service/project.go]: done

**Pull request merged**